src/Application/PrizeDraw/Queries/Mapping.cs

```diff
@@ -31,7 +28,7 @@ public Mapping()
                     .Where(ua => ua.AwardedAt.Year == DateTime.Now.Year && ua.AwardedAt.Month == DateTime.Now.Month && ua.AwardedAt.Day == DateTime.Now.Day)
                     .Sum(ua => ua.Achievement.Value)))
                 .ForMember(dst => dst.PointsThisWeek, opt => opt.MapFrom(src => src.UserAchievements
-                    .Where(ua => start <= ua.AwardedAt && ua.AwardedAt <= end)
+                    .Where(ua => DateTime.Now.AddDays(-7) <= ua.AwardedAt && ua.AwardedAt <= DateTime.Now)
```

Comment:
I'm not going to block this because we're doing it everywhere (and I'm responsible for it) but we should switch to using `IDatetimeProvider`. See #538 - we should probably add that to the next sprint to avoid cumulative technical debt.